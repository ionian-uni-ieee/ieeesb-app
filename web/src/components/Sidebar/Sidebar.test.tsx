import React from 'react'

import { render } from '@testing-library/react'

import Sidebar from './Sidebar'

describe('Sidebar', () => {
  it('should render', () => {
    const { container } = render(<Sidebar />)
    expect(container).toMatchSnapshot()
  })
})
