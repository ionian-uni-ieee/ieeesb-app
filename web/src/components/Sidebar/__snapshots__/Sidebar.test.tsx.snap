// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar should render 1`] = `
<div>
  <div
    class="
        sidebar
        undefined
      "
  >
    <span
      class="
        logo
        
      "
    >
      <svg
        style="height: 38px;"
        viewBox="0 0 150 38"
        width="fit-content"
      >
        logo_white.svg
      </svg>
    </span>
    <hr
      class="sidebar__divider"
    />
  </div>
</div>
`;
